regression/pages/studio/pages_page_studio.py

```diff
@@ -143,3 +144,67 @@ def click_and_verify_see_an_example(self):
         self.wait_for_element_visibility(
             'img[alt="Preview of Pages in your course"]', 'Pop up visibility'
         )
+
+    def click_hide_show_toggle(self, index=0):
+        """
+        Clicks hide/show toggle button
+        """
+        toggle_checkbox_css = '.toggle-checkbox'
+        toggle_checkbox = self.q(css=toggle_checkbox_css).results[index]
+        ActionChains(self.browser).move_to_element(
+            toggle_checkbox
+        ).click().perform()
+        wait_for_notification(self)
+        return self.browser.execute_script(
+            "return $('{}:eq({})').parents().eq(3).find("
+            "'.course-nav-item-header .title').text()".format(
+                toggle_checkbox_css, index
+            )
+        )
+
+    def get_movable_default_pages_order(self):
+        """
+        Returns the order of moveable default pages
+        """
+        return self.q(
+            css='.course-nav-item.is-movable .course-nav-item-header'
+        ).text
+
+    def get_moveable_default_pages(self):
+        """
+        Returns pages which are default and moveable
+        """
+        return self.q(
+            css='.course-nav-item.is-movable .drag-handle.action'
+        ).results
+
+    def is_page_shown(self, index=0):
+        """
+        Check whether a page is shown or not at a given index
```

Comment:
I'm confused...I don't see any `wait_for` kinds of methods in here. When you say "whether a page is shown or not" do you really mean "whether a page is *configured to* show or not?"